<launch>
  <master auto="start"/>
  <param name="/use_sim_time" value="true"/>

<remap from="scan" to="base_scan"/>


<node pkg="mr_exploration" name="slam" type="slam_node" output="screen" args="-timeUpdate 0 -angularUpdate 0.5 -linearUpdate 0.5 -scanTopic scan -type 0" />


<!--
<include file="$(find mr_exploration)/move_base_config/slam_gmapping.xml"/>
-->


<!--
<node pkg="mr_exploration" name="planner" type="frontier_planner" output="screen" args="-iter 1"  />
-->

<node pkg="stage_ros" type="stageros" name="stageros" args="$(find mr_exploration)/stagefiles/hospital.world" respawn="false" >
<param name="base_watchdog_timeout" value="0.2"/>
</node>


<node pkg="rviz" type="rviz" name="visualizer" output="screen" args="-d $(find mr_exploration)/config.rviz"/>


<include file="$(find mr_exploration)/move_base_config/move_base_stage.xml"/>



</launch>

